q2_diversity/plugin_setup.py

```diff
@@ -711,3 +711,37 @@
         citations['pearson1895note'],
         citations['spearman1904proof']]
 )
+
+plugin.visualizers.register_function(
+    function=q2_diversity.adonis,
+    inputs={'distance_matrix': DistanceMatrix},
+    parameters={'metadata': Metadata,
+                'formula': Str,
+                'permutations': Int % Range(1, None)
+                'n_jobs': Int % Range(0, None)},
```

Comment:
Sorry, messed this up the first time when I suggested:
```suggestion
                'n_jobs': Int % Range(1, None)},
```